src/devices/Bmp180/CalibrationData.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Bmp180
+{
+    internal class CalibrationData
+    {
+        public short AC1 {get;set;}
+        public short AC2 {get;set;}
+        public short AC3 {get;set;}
+        public ushort AC4 {get;set;}
+        public ushort AC5 {get;set;}
+        public ushort AC6 {get;set;}
+
+        public short B1 {get;set;}
+        public short B2 {get;set;}
+
+        public short MB {get;set;}
+        public short MC {get;set;}
+        public short MD {get;set;}
+
+        internal void ReadFromDevice(Bmp180 bmp180)
+        {
+            AC1 = (short)bmp180.Read16BitsFromRegisterBE((byte)Register.AC1);

```

Comment:
**Question:** Are the casts to byte necessary? The enum is backed by a byte already.

Reply:
If I do not, I have the following error code.

[Compiler Error CS0266](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0266)

Even the first example of this guide uses Cast (byte)

[Enumeration types](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/enumeration-types)